BookingSystem/Frontend/src/hotelSearch/components/HotelSearch.js

```diff
@@ -1,19 +1,52 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import HotelList from "./HotelList";
+import SearchForm from "./SearchForm";
 
-class HotelSearch extends Component {
-  render() {
-    const { hotels, getHotels } = this.props;
-    return (
-      <div>
-        <HotelList hotels={hotels} />
-        <button onClick={() => getHotels("query")}>Get hotels</button>
-      </div>
-    );
-  }
-}
-
+const HotelSearch = props => {
+  const {
+    startDate,
+    endDate,
+    roomSize,
+    country,
+    city,
+    countryOptions,
+    cityOptions,
+    handleSubmit,
+    handleReset,
+    handleInputChange,
+    handleStartDateChange,
+    handleEndDateChange,
+    handleCountryOptionsChange,
+    handleCityOptionsChange,
+    handleCountryChange,
+    handleCityChange,
+    hotels
+  } = props;
+  return (
+    <div>
+      <SearchForm
+        startDate={startDate}
```

Comment:
you may consider uniting search options into one object and passing it down to components (spread ... is also applicable here, but be careful https://reactjs.org/docs/jsx-in-depth.html#spread-attributes)